config/runtime/tests/rt-tests.jinja2

```diff
@@ -25,7 +25,7 @@
         PRIORITY: {{ priority|default('98') }}
         THREADS: {{ threads|default('2') }}
       {% elif tst_cmd == 'pi-stress' %}
-        MLOCKALL: {{ mlockall|default('true') }}
+        MLOCKALL: {{ mlockall|default("true") }}
```

Comment:
```suggestion
        MLOCKALL: {{ mlockall|default('"true"') }}
```

Verified with:
```
$ ./kci job generate --api staging -c ../rampup/kernelci-pipeline/config --runtime=lava-collabora 66ec2b36ebd9a94074b9dfc9 | lavacli jobs submit -
```
[15790058](https://lava.collabora.dev/scheduler/job/15790058)

Why?
1. First set of quotations allows Jinja template engine to treat it as string
2. Second set is for LAVA to also treat is as string and prevent YAML parser from making it a bool

Why does it even matter?
1. Test runner script [expects command name to run in a condition check](https://github.com/Linaro/test-definitions/blob/6c6ed7499e40c29da7dabfa2149350d13fb6496d/automated/linux/pi-stress/pi-stress.sh#L37)
2. This test runner script should be rewritten so that it converts truthy/falsy arguments and doesn't allow external commands to control its flow - it's a vector for command injection from bad actors

Bottom line:
1. Double quotations will allow this suite to be executed
2. Long-term solution would be fixing upstream test runner script

Reply:
@pawiecz Thanks a lot for taking a look and fixing it. I couldn't figure out the parsing.

Created: https://github.com/Linaro/test-definitions/issues/532